docs/DeveloperGuide.md

```diff
@@ -284,11 +284,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​                                       | I want to …​                                                       | So that I can…​                               |
 |----------|-----------------------------------------------|--------------------------------------------------------------------|-----------------------------------------------|
-| `* * *`  | new user                                      | create new entry                                                   | store the connections and internship listings |
-| `* * *`  | user keeping track of my internship positions | modify the details of the internship positions                     | be aware of the status                        |
-| `* * *`  | user keeping track of many connections        | to be able to modify the details of these connections              | ensure the information remains up-to-date     |
+| `* * *`  | new user                                      | create new internship entry                                        | store the connections and internship listings |
```

Comment:
I believe it will be better if we standardize plural or singular throughout ("entry" vs "entries")